dumb-init.c

```diff
@@ -199,6 +257,9 @@ char **parse_command(int argc, char *argv[]) {
             case 'r':
                 parse_rewrite_signum(optarg);
                 break;
+            case 'b':
+                survive_bereaving = 1;
+                break;
             default:
                 exit(1);
         }
```

Comment:
Since `survive_bereaving` only works properly when dumb-init is running as PID 1, I think it'd be a good idea to add a check like:

```c
if (survive_bereaving && getpid() != 1) {
    PRINTERR("--survive-bereaving is only valid when dumb-init is running as PID 1.\n");
    exit(1);
}
```

I think that would reduce confusion for anybody who tries to use the feature without being PID1.